docs/data-sources/fabric_connection.md

```diff
@@ -129,14 +123,13 @@ Read-Only:
 - `service_token` (Set of Object) (see [below for nested schema](#nestedobjatt--a_side--service_token))
 
 <a id="nestedobjatt--a_side--access_point"></a>
-
 ### Nested Schema for `a_side.access_point`
 
 Read-Only:
 
 - `account` (Set of Object) (see [below for nested schema](#nestedobjatt--a_side--access_point--account))
 - `authentication_key` (String)
-- `gateway` (Set of Object, Deprecated) **Deprecated** `gateway` Use `router` attribute instead (see [below for nested schema](#nestedobjatt--a_side--access_point--gateway))
```

Comment:
we lose a documented deprecation flag. Is the field not marked as deprecated? (also in z_side docs)

Reply:
It is marked appropriately, but because it is a nested field the descriptions and further details aren't picked up by `tfplugindocs`. Just the name of the attribute and the type.

Reply:
Actually, there may be another reason for this. The data_source isn't showing descriptions for any of the nested attributes even though the resource schema displays them. Might be because of the way we're changing attribute behavior to computed and then setting the data_source value.

Resource attributes are showing nested descriptions.